### PR TITLE
Add contact form

### LINKED ISSUE #13: Add Contact Form
# Suggestion/Improvement

🌟 Use react.js to create a form component.

## Description

Create a contact form that can be edited by the project users according to their requirements. Add the form to the contact page linked to the contact menu item in the header/ navigation.

## Motivation

With all the other necessary components in the starter template available, adding a contact form will make it even better.

## Implementation (if applicable)

Make the form accessible, responsive and well-styled. Include a screenshot in the pull request.

### PR DESCRIPTION
**Pull request by ToobaJamal**
## Assigned Issue

- [x] I am assigned to the related issue below.
Closes/ fixes #13 

## Description
This PR adds a contact form component in the starter template which is displayed on the screen when the contact menu item is clicked.
This PR also fixes footer positioning which was appearing on top of the other content. Removing absolute position and applying display flex fixed the issue.

#### Before footer position fix
shesharp

#### After footer position fix
Shesharp1

#### Contact form on desktop
Shesharp2

#### Contact form on mobile
Shesharp3

## Type of PR

- [x] ✨ Feature
- [x] 🐞 Bug fix

### CODE REVIEW

File: src/components/ContactForm.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+
+// Header component with logo and navigation links
```
Comment (ghost) [inline src/components/ContactForm.jsx]: hi @ToobaJamal , can you update this comment to reflect the component? The other comments are great!

File: package-lock.json [hunk 2]
```diff
@@ -16504,6 +16593,19 @@
         "is-typedarray": "^1.0.0"
       }
     },
+    "node_modules/typescript": {
```
Comment (ghost) [inline package-lock.json]: do we need this module? I think we have only JavaScript in this project.

File: package-lock.json [hunk 3]
```diff
@@ -3426,6 +3426,95 @@
...
+    "node_modules/@testing-library/dom/node_modules/has-flag": {
+      "version": "4.0.0",
+      "resolved": "https://registry.npmjs.org/has-flag/-/has-flag-4.0.0.tgz",
+      "integrity": "sha512-EykJT/Q1KjTWctppgIAgfSO0tKVuZUjhgMr17kqTumMl6Afv3EISleU7qZUzoXDFTAHTDC4NOoG/ZxU3EvlMPQ==",
+      "peer": true,
+      "engines": {
+        "node": ">=8"
+      }
+    },
+    "node_modules/@testing-library/dom/node_modules/supports-color": {
```
Comment (ghost) [inline package-lock.json]: Also wondering, what is the purpose of these packages? Are they needed for the project to run? We are trying to keep the number of dependencies minimal. :)

**Review by ghost — CHANGES_REQUESTED** [review]
Thanks for your PR @ToobaJamal , the screenshots look great. I'm just wondering about the packages that have been added, and if you can update the comment that would be great!

**Comment by ToobaJamal** [comment]
> Thanks for your PR @ToobaJamal , the screenshots look great. I'm just wondering about the packages that have been added, and if you can update the comment that would be great!

@mariahlaqua, this is strange. I just forked the repo, run the npm install, and added my code. I'll fix it on my end and update you.

File: package-lock.json (hunk 3 above)
Comment (ToobaJamal) [reply package-lock.json]: @mariahlaqua, I have fixed all the issues.
Lesson learned today: always run `git status` before `git add .`

**Review by ghost — APPROVED** [review]
No worries @ToobaJamal , I noticed that these are coming in and out of the package, it might be a global installation or something 🤔 ?

I'm not sure! Regardless, appreciate you making the time to change it. Thanks again for creating this issue and pull request!!

**Pull request merged**